gradle/libs.versions.toml

```diff
@@ -1,52 +1,68 @@
 [versions]
+kotlin = "1.7.10"
+
 ## SDK Versions
 minSdk = "21"
-targetSdk = "31"
-compileSdk = "31"
+targetSdk = "33"
+compileSdk = "33"
 
 # Dependencies
-compose = "1.1.1"
-composeCompiler = "1.2.0-alpha08"
-coroutines = "1.6.1"
-multiplatformSettings = "0.9"
-koin = "3.1.6"
-kotlin = "1.6.20"
-ktor = "2.0.0"
-lifecycle = "2.4.1"
+android-gradle-plugin = "7.2.2" # Don't bump to 7.3.x until Intellij supports it (unless you're ok with only Android Studio)
+ktlint-gradle = "11.0.0"
+gradle-versions = "0.42.0"
+
+compose = "1.2.1"
+composeCompiler = "1.3.0"
+
+android-desugaring = "1.1.8" # Don't bump to 1.2.x until AGP is 7.3.x
+androidx-core = "1.8.0"
+androidx-test-junit = "1.1.3"
+androidx-activity-compose = "1.5.1"
+androidx-lifecycle = "2.5.1"
+
+junit = "4.13.2"
+
+coroutines = "1.6.4"
+kotlinx-datetime = "0.4.0"
+ktor = "2.1.1"
+
+robolectric = "4.8.2"
+
+kermit = "1.1.3"
+stately = "1.2.3"
+
+accompanist-swiperefresh = "0.25.1"
+koin = "3.2.0"
+multiplatformSettings = "1.0.0-alpha01"
+turbine = "0.8.0"
 sqlDelight = "1.5.3"
 
 [libraries]
```

Comment:
I pulled out all of the inline versions in the library declarations to be their own version declarations, so it's easier to see them all when updating.